.github/workflows/release.yml

```
@@ -0,0 +1,29 @@
+on:
+  push:
+    tags:
+      - "*"
```

Let's change the event listener temporarily just so we can see this run before merging.

```suggestion
on:
  pull_request:
    branches: [ master ]
```

We can change it back after.